gems/pending/util/esx_thumb_print.rb

```diff
@@ -0,0 +1,34 @@
+require 'util/thumb_print'
+
+class ESXThumbPrint < ThumbPrint
+  attr_reader :user, :password, :cert
+
+  def initialize(host, user, password)
+    @user     = user
+    @password = password
+    super(host)
+  end
+
+  def uri
+    url  = "https://" + @host + "/host/ssl_cert"
+    @uri = URI(url)
+    @uri
+  end
+
+  def http_request
+    super
```

Comment:
Kinda yes and kinda no - I'm ignoring the return but the @http variable is set in the base class and used by both the vcenter and esx classes